docs/serverless/08-05-create-git-function.md

```diff
@@ -0,0 +1,183 @@
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: serverless.kyma-project.io/v1alpha1
+    kind: Function
+    metadata:
+      name: $GIT_FUNCTION
+      namespace: $NAMESPACE
+    spec:
+      type: git
+      runtime: nodejs12
+      source: $GIT_FUNCTION
+      reference: master
+      baseDir: orders-service/function
+    EOF
+    ```
+
+    >**NOTE:** See this [Function's code and dependencies](https://github.com/kyma-project/examples/tree/master/orders-service/function).
+
+5. Check if your Function was created and all conditions are set to `True`:
+
+    ```bash
+    kubectl get functions $GIT_FUNCTION -n $NAMESPACE
+    ```
+
+    You should get a result similar to the following example:
+
+    ```bash
+    NAME            CONFIGURED   BUILT     RUNNING   RUNTIME    VERSION   AGE
+    test-function   True         True      True      nodejs12   1         96s
+    ```
+
+    </details>
+    <details>
+    <summary label="console-ui">
+    Console UI
+    </summary>
+
+1. Create a Namespace or select one from the drop-down list in the top navigation panel.
+
+2. Create a Secret (optional).
+
+  If you use a secured repository, you must first create a Secret with either basic (username and password or token) or SSH key authentication to this repository in the same Namespace as the Function. To do that, follow these sub-steps:
+
+  - On your machine, create this YAML file with one of these Secret definitions:
+
+    - Basic authentication:
+
+      ```yaml
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: git-creds-basic
+        namespace: {NAMESPACE}
+      type: Opaque
+      data:
+        username: {USERNAME}
+        password: {PASSWORD_OR_TOKEN}
+      ```
+
+    - SSH key:
+
+
+      ```yaml
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: git-creds-key
+        namespace: {NAMESPACE}
+      type: Opaque
+      data:
+        key: {SSH_KEY}
+      ```
+
+    >**NOTE:** Read more about the [supported authentication methods](#details-git-source-type).
+
+  - Go to your Namespace view and select **Deploy new resource**.
+
+  - Locate the YAML file with the Secret and select **Deploy**.
+
+3. In the left navigation panel, go to **Development** > **Functions** and select the **Repositories** tab.
+
+4. Select **Connect Repository**, fill in the **Url** field with `https://github.com/kyma-project/examples.git`, and confirm by selecting **Connect**.
+
+    >**NOTE:** If you want to connect a secured repository, change the **Authorization** field from `Public` to `Basic` or `SSH key` and fill in the required fields.
+
+5. Go to the **Functions** tab and select **Create Function**.
+
+6. In the pop-up box, change `Source Type` to `From Repository`, select created Repository's name, fill in the `Reference` field with `master` and `Base Directory` field with `orders-service/function` values, and select **Create** to confirm changes.
```

Comment:
```suggestion
6. In the pop-up box, change `Source Type` to `From Repository`. Select the created repository's name and fill in the `Reference` field with `master` and `Base Directory` field with `orders-service/function` values. Select **Create** to confirm changes.
```

Reply:
I'd divide this long sentence into three shorter ones to make the whole point more digestible 😉